3rdparty/mbedtls/CMakeLists.txt

```diff
@@ -44,9 +45,18 @@ endif ()
 list(JOIN MBEDTLS_WRAP_CFLAGS_LIST " " MBEDTLS_WRAP_CFLAGS)
 configure_file(config.h config.h) # This copies from source to binary folders.
 
+# Since we define mbedtls to use an alternate entropy source, it uses an
+# undefined mebdtls_hardware_poll function. We define it here and add it to
+# the mbedcrypto library, so that we avoid link errors and also circular
+# library dependecies.
```

Comment:
FYI, this file gets entirely rewritten in a future PR since imported library targets don't work properly in cmake with install targets.

Reply:
I filed issue #1836